modules/resources/resources.py

```diff
@@ -112,4 +114,51 @@ def generate_static_pages():
                     md_file.write(content)
             else:
                 with open(os.path.join(site_config.resources_markdown_path, static_page), "w", encoding='utf8') as md_file:
-                    md_file.write(content)
\ No newline at end of file
+                    md_file.write(content)
+
+def generate_working_with_attack():
+    """ Responsible for generating working with ATT&CK and creating excel files
+    """
+
+    excel_dirs = [
+        f"enterprise-attack-{site_config.full_attack_version}", 
+        f"mobile-attack-{site_config.full_attack_version}",
+        f"ics-attack-{site_config.full_attack_version}"
+    ]
+    files_types = ["matrices", "mitigations", "relationships", "software", "tactics", "techniques"]
+
+    # Verify if directories exists
+    if not os.path.isdir(site_config.web_directory):
+        os.makedirs(site_config.web_directory)
+    
+    resources_dir = os.path.join(site_config.web_directory, 'resources')
+    if not os.path.isdir(resources_dir):
+        os.makedirs(resources_dir)
+    
+    working_with_attack = os.path.join(resources_dir, 'working-with-attack')
+    if not os.path.isdir(resources_dir):
+        os.makedirs(working_with_attack)
+
+    attackToExcel.export("enterprise-attack", site_config.full_attack_version, working_with_attack)
+    attackToExcel.export("mobile-attack", site_config.full_attack_version, working_with_attack)
+    attackToExcel.export("ics-attack", site_config.full_attack_version, working_with_attack)
```

Comment:
It would be preferable if these could be hosted in the `docs` folder instead of `/resources/working-with-attack/{domain-version}/{domain-version-type}.xlsx`. While they are documents hosted by working with ATT&CK, they structurally belong in the docs folder alongside our other website documents.

Reply:
docs output folder for reference: https://github.com/mitre-attack/attack-website/tree/gh-pages/docs